scripts/document.sh

```diff
@@ -30,7 +30,7 @@ BASE_URL="https://www.mapbox.com/mapbox-navigation-ios"
 
 # Link to directions documentation
 DIRECTIONS_VERSION=$(grep 'MapboxDirections.swift' Cartfile.resolved | grep -oE '"v.+?"' | grep -oE '[^"v]+')
-DIRECTIONS_SYMBOLS="Directions|DirectionsOptions|DirectionsResult|Intersection|Lane|Match|MatchOptions|Route|RouteLeg|RouteOptions|RouteStep|SpokenInstruction|Tracepoint|VisualInstruction|VisualInstructionComponent|Waypoint"
+DIRECTIONS_SYMBOLS="Directions|DirectionsOptions|DirectionsResult|Intersection|Lane|Match|MatchOptions|Route|RouteLeg|RouteOptions|RouteStep|SpokenInstruction|Tracepoint|VisualInstruction|VisualInstructionComponent|Waypoint|RoadClasses|ComponentRepresentable"
```

Comment:
Also VisualInstructionBanner. Are we missing anything else, @vincethecoder?

Reply:
@1ec5 ... and `LaneIndicationComponent`

Reply:
This list is meant for symbols that can appear in the public API, for example as argument types. It looks like `LaneIndicationComponent` won’t be exposed anywhere in the public API even after #1514.